ext/src/ruby_api/execution_error.rs

```diff
@@ -39,6 +41,13 @@ impl ExecutionError {
         self.path.clone()
     }
 
+    pub fn to_h(&self) -> RHash {
```

Comment:
I think this can remain private initially as it's not used anywhere outside of this file
```suggestion
    fn to_h(&self) -> RHash {
```

Reply:
Got it -- at first, I was thinking it had to be `pub` in order to be attached the the Ruby class, but I see that's not the case for `inspect` further down :+1: